src/main/java/mate/academy/spring/config/AppConfig.java

```diff
@@ -44,4 +46,9 @@ public LocalSessionFactoryBean getSessionFactory() {
         factoryBean.setPackagesToScan("mate.academy.spring.model");
         return factoryBean;
     }
+
+    @Bean
+    public PasswordEncoder getEncoder() {
```

Comment:
basically, when you use the `@Bean` annotation, the method's name becomes the bean name, 

so just a suggestion, to name @Bean methods like if you name a bean.